HOW-TO.md

```diff
@@ -3,10 +3,10 @@ You must have an active CyberGRX account that has the ability to manage users.
 
 The workflow is pretty simple:
  - Enter `Manage my company user accounts` using the top right navigational element.
- - Click the tab to `Manage Access Tokens`
+ - Click the tab to `Access Tokens`
  - Click the button to `Add a new token`
  - Accept the promt creating a token.
- - Show the secret and copy it, this is the only time that this token will be available over the API!
+ - Click `view the secret` the secret and copy it, this is the only time that this token will be available over the API!
```

Comment:
Click `view the secret` to view the secret and then copy it, this is the only time that this token will be available over the API!

Reply:
@miriamgrigsby Fixed!